products/rhel10/profiles/default.profile

```diff
@@ -20,3 +20,4 @@ selections:
   - configure_tmux_lock_after_time
   - configure_tmux_lock_command
   - configure_tmux_lock_keybinding
+  - sysctl_user_max_user_namespaces
```

Comment:
This shouldn't be added to the default profile because it's part of OSPP profile. The OSPP is also a hidden profile, same as the default profile.